docs/kap_proposals/kap_1_external_dependencies.md

```diff
@@ -0,0 +1,34 @@
+# External dependencies
+This features allows kapitan to fetch files from online repositories/sources during compile and store in a particular target directory.
+
+Author @yoshi-1224
+
+## Specification
+Specify the files to be fetched as follows:
```

Comment:
Add newlines after headings and after text/code blocks. Some markdown parsers have issues parsing them otherwise.